vroom/zprint.vroom

```diff
@@ -0,0 +1,95 @@
+The zprint formatter knows how to format Clojure.
+If you aren't familiar with basic codefmt usage yet, see main.vroom first.
+
+We'll set up codefmt and configure the vroom environment, then jump into some
+examples.
+
+  :source $VROOMDIR/setupvroom.vim
+
+  :let g:repeat_calls = []
+  :function FakeRepeat(...)<CR>
+  |  call add(g:repeat_calls, a:000)<CR>
+  :endfunction
+  :call maktaba#test#Override('repeat#set', 'FakeRepeat')
+
+  :call codefmt#SetWhetherToPerformIsAvailableChecksForTesting(0)
+
+The zprint formatter expects the zprint executable to be installed on your system.
+
+  @clear
+  % (defn x [] (cond nil 1 :else 2))
+
+  :FormatCode zprint
+  ! cd .* zprint .*
+  $ (defn x
+  $   []
+  $   (cond nil 1
+  $         :else 2))
+
+The name or path of the zprint executable can be configured via the
+zprint_executable flag if the default of "zprint" doesn't work.
+
+  @clear
+  % (defn x [] (cond nil 1 :else 2))
+
+  :Glaive codefmt zprint_executable='/usr/local/bin/zprint'
+  :FormatCode zprint
+  ! cd .* /usr/local/bin/zprint .*
+  $ (defn x
+  $   []
+  $   (cond nil 1
+  $         :else 2))
+  :Glaive codefmt zprint_executable='zprint'
+
+You can format any buffer with zprint specifying the formatter explicitly.
+
+  @clear
+  % (defn x [] (cond nil 1 :else 2))
+
+  :FormatCode zprint
+  ! cd .* zprint .*
+  (defn x
+    []
+    (cond nil 1
+          :else 2))
+  @end
+
+It can format specific line ranges of code using :FormatLines.
+
+  @clear
+  % (defn x [] (cond nil 1 :else 2))<CR>
+  |(defn x [] (cond nil 1 :else 2))
+
+  :1,1FormatLines zprint
+  ! cd .* zprint .*
+  $ (defn x
+  $   []
+  $   (cond nil 1
+  $         :else 2))
+  (defn x
+    []
+    (cond nil 1
+          :else 2))
+  (defn x [] (cond nil 1 :else 2))
+  @end
+
+It is also the default formatter for the clojure file type.
```

Comment:
The descriptions would read better if you reordered the sections so this `It is also the default formatter for` section is right after the `You can format any buffer` section.

Reply:
Will do

Reply:
I didn't reorder exactly like that, but I reworked the narrative so it's clearer and doesn't have a continuity dependency between the sections. (I wanted to keep `:FormatCode` and `:FormatLines` next to each other, but previously the test for `:FormatCode` was worded to emphasize calling zprint explicitly.)